Makefile.ci.toml

```diff
@@ -10,11 +10,13 @@ args = ["check", "--locked", "--workspace"]
 [tasks.ci-check-wasm]
 category = "CI - CHECK"
 command = "cargo"
+env = { RUSTFLAGS = "--cfg surrealdb_unstable" }
 args = ["check", "--locked", "--package", "surrealdb", "--features", "protocol-ws,protocol-http,kv-mem,kv-indxdb,http,jwks", "--target", "wasm32-unknown-unknown"]
 
 [tasks.ci-clippy]
 category = "CI - CHECK"
 command = "cargo"
+env = { RUSTFLAGS = "--cfg surrealdb_unstable" }
```

Comment:
Why do we need `surrealdb_unstable` for tests other than `test-experimental-parser`?

Reply:
I think a lot of development going forward is going to be against `sql2` so I think it's a good idea to test those as well, otherwise we will end up merging a lot of untested code. In this particular case, we need the flag because `ci-clippy` activates `jwks` which is only available on `sql2` so far.

Reply:
To be extra clear, it's not just the experimental parser that's unstable currently. `sql2` is too. I think we will stabilise `sql2` very soon though.